bot/start.ts

```diff
@@ -495,10 +496,9 @@ const initialize = (botToken: string, options: Partial<Telegraf.Options<Communit
       const order = await Order.findOne({ _id: orderId });
       if (order === null) return;
 
-       // Check if the order status is already PAID_HOLD_INVOICE
-      if (order.status === 'PAID_HOLD_INVOICE') {
-      await ctx.reply(ctx.i18n.t('order_already_settled'));
-      return;
+      // Check if order status is DISPUTE
```

Comment:
@Catrya let's make this comment more meaningful? let's explain the why instead of the what